### ISSUE TITLE
experience on the hyperparameter of the loss

### ISSUE THREAD

**Issue by cxmscb** [wangch-g/lanet#7]
Hi, I retrained the lanet model on one new dataset, and found that the numbers of the detected points is small. The models does not detected the points when the images are corrupted. Could I change the hyperparameter for improving the numbers of the detected points?

**Comment by cxmscb** [comment]
I read the following code in the loss function. Could it hurt the performance?
`if n_feat < 20:
                continue`

**Comment by wangch-g** [comment]
I suggest you finetuning the pre-trained model on the new dataset.
Could you show some images in the dataset?

**Comment by cxmscb** [comment]
Thanks. I found these two images hard to match, and I have no idea to deal with it.
[img1](https://upload.wikimedia.org/wikipedia/commons/thumb/6/6d/Flag_of_the_Chinese_Communist_Party.svg/255px-Flag_of_the_Chinese_Communist_Party.svg.png)
[img2](https://www.women.org.cn/zhuanti/www.women.org.cn/newsfl/upfile/newsimg_150149.jpg)

**Comment by cxmscb** [comment]
I change the weight of the consistent-constrain loss, which can help the repeatness.

`repeatable_constrain + 1.5 * consistent_constrain + aware_consistent_loss`
